docs/users/virtualsites.md

```diff
@@ -104,42 +113,41 @@ site parameters. Let us consider 4-, 5-, and 6-point water models:
-  different indexed atoms (parents), causing the energy to be different than
-  the non-virtual site parameterization. In the future, the
-  `exclusion_policy="local"` will account for this, and make virtual sites that
-  share at least one "parent" atom not interact with each other. As a special
-  note: when applying a `MonovalentLonePair` to a completely symmetric
-  molecule, e.g. water, `all_permutations` can come into play, but this will
-  apply two particles (one for each hydrogen).
-
-Finally, the toolkit handles the organization of atoms and virtual sites in a
-specific manner. Virtual sites are expected to be added *after all molecules in
-the topology are present*. This is because the Open Force Field Toolkit
-organizes a topology by placing all atoms first, then all virtual sites last.
-This differs from the OpenMM Modeller object, for example, which interleaves
-the order of atoms and virtual sites in such a way that all particles of a
-molecule are contiguous. In addition, due to the fact that a virtual site may
-contain multiple particles coupled to single parameters, the toolkit makes a
-distinction between a virtual *site*, and a virtual *particle*. A virtual site
-may represent multiple virtual particles, so the total number of particles
-cannot be directly determined by simply summing the number of atoms and virtual
+  If both had the same name, then the typical SMIRNOFF hierarchy rules are
+  used, and only the last matched parameter would be applied.
+
+## Ordering of atoms and virtual sites
+
+The toolkit handles the orders the atoms and virtual sites in a topology in a
+specific manner for internal convenience. 
+
+Virtual sites are expected to be added *after all molecules in the topology are present*. 
+This is because the Open Force Field Toolkit organizes a topology by placing all atoms 
+first, then all virtual sites last. This differs from the OpenMM Modeller object, for 
+example, which interleaves the order of atoms and virtual sites in such a way that all 
+particles of a molecule are contiguous. 
+
+In addition, due to the fact that a virtual site may contain multiple particles coupled 
+to single parameters, the toolkit makes a distinction between a virtual *site*, and a virtual 
+*particle*. A virtual site may represent multiple virtual particles, so the total number of 
+particles cannot be directly determined by simply summing the number of atoms and virtual
 sites in a molecule. This is taken into account, however, and the 
 [`Molecule`](openff.toolkit.topology.Molecule) and
 [`Topology`](openff.toolkit.topology.Topology) classes both implement `particle`
 iterators.
 
+:::{note}
+The distinction between a virtual site and virtual particle is due to be removed in a future
+version of the toolkit, and a 'virtual site' will simply refer to one massless particle placed
+on a parent atom rather than to a collection of massless particles.
```

Comment:
This means that in the future, an API will offer access to VSites, and never make mention of VParticles. If two VSites (new definition) are actually based on the same FF parameter, the user can tell this by seeing that they have the same name and parent atom.